README.md

```diff
@@ -123,31 +135,10 @@ Path:       /home/linuxbrew/.linuxbrew/Cellar/python@3.12/3.12.5
 Executable: /home/linuxbrew/.linuxbrew/Cellar/python@3.12/3.12.5/bin/python3.12
 ```
 
-Now this virtual environment needs to be removed, because the path to this folder vary from machine and we need debugger to access these virtual environment in more consistent way. 
-
-
-Remove the virtual env by running:
-> Note: If you don't see any path listed under Virtualenv, you can skip this step.
+Create a settings.json file to point the POETRY_ENV_INFO to the Executable directory in the Virtualenv
+```json
+{"POETRY_ENV_INFO": "/home/ilhanbae/.cache/pypoetry/virtualenvs/app-6zZbMuyK-py3.12/bin/python"}
```

Comment:
That doesn't seem right... why do we have a person-specific path?

Also, we could just use a vscode setting rather than an environment variable, like i have in my pr. since this is all for vscode that would seem more appropriate?

Reply:
poetry installs the python executable in a person-specific path.

what the README is saying is "run poetry env info and use that in your launch.json".  the changes are in settings.json, as your PR recommends.

What's missing from your PR is where the python path comes from